tests/pipeline-all-amps/repo/Dockerfile

```diff
@@ -44,5 +40,14 @@ RUN chgrp -R Alfresco ${TOMCAT_DIR} && \
   chmod -R g+r ${TOMCAT_DIR}/webapps
 
 #Use the alfresco user
-USER alfresco
+#USER alfresco
```

Comment:
This can be deleted

Reply:
ideally I would like to put it back, but I haven't understood why the alfresco-mmt needs write access outside of the target webapp